app/src/marketplace/components/PriceField/index.jsx

```diff
@@ -35,21 +35,20 @@ const PriceField = ({
                     [styles.withError]: !!hasError,
                 })}
             >
-                <TextControl
-                    immediateCommit={false}
-                    commitEmpty
+                <Text
+                    smartCommit
                     selectAllOnFocus
                     value={value}
                     className={styles.input}
                     {...inputProps}
                 />
                 <span className={styles.currency}>{currency}</span>
             </div>
-            <InputError
-                eligible={hasError}
-                message={lastError}
-                preserved={false}
-            />
+            {hasError && !!lastError && (
+                <FormControlErrors overlap theme={MarketplaceTheme}>
```

Comment:
[Themes](https://styled-components.com/docs/advanced#theming) is something I'm figuring out. Pretty neat feature at this point. Wonder if we can use it more often.